app/lib/features/activities/widgets/notification_card.dart

```diff
@@ -59,22 +55,20 @@ class NotificationCard extends ConsumerWidget {
           },
         );
         room = Consumer(
-          builder: (ctx, ref, child) {
+          builder: (context, ref, child) {
             final spaceProfile = ref.watch(spaceProfileDataProvider(space));
             return spaceProfile.when(
               data: (value) => Text(value.displayName ?? roomId),
-              error: (error, stackTrace) {
-                // FIXME: fallback would be nice
-                return Text('Failed to load space due to $error');
-              },
+              error: (error, stackTrace) =>
+                  Text('Failed to load space Text(roomId)due to $error'),
               loading: () => Text(roomId),
             );
           },
         );
       } else {
         final convo = notification.convo()!;
         avatar = Consumer(
-          builder: (ctx, ref, child) {
+          builder: (context, ref, child) {
```

Comment:
Actually I renamed inner `context` to `ctx` so that it can be distinguished with outer `context`.